mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -66,6 +66,8 @@ CLASS net/minecraft/class_31 net/minecraft/item/ItemStack
 		ARG 2 attacker
 	METHOD method_706 writeNbt (Lnet/minecraft/class_8;)Lnet/minecraft/class_8;
 		ARG 1 nbt
+	METHOD method_707 getAttackDamage (Lnet/minecraft/class_57;)I
+		ARG 1 otherEntity
```

Comment:
This name is confusing, what is the other entity being used for?

Reply:
the other entity is the one being attack, altho i agree that maybe attackedEntity or attacked might be a more descriptive name, im not sure if Yarn has any name like this. There is getAttackDamage in SwordItem but it has not parameters

Reply:
Renamed to attackedEntity

Reply:
Similar use case in yarn has "target" as the name - https://github.com/FabricMC/yarn/blob/bd3e21e1cd577dc1b0f4ec41cc88d8933e78c7e1/mappings/net/minecraft/item/ItemStack.mapping#L640